js/index.js

```diff
@@ -37,6 +42,104 @@ const siteContent = {
+navItem[4].textContent = siteContent["nav"]["nav-item-5"];
+navItem[5].textContent = siteContent["nav"]["nav-item-6"];
+
+
+
+
+const firstContent = document.createElement('a')
+firstContent.textContent = "Energy";
+
+const lastContent = document.createElement('a')
+lastContent.textContent = "Sustainable";
+
+const parentNav = document.querySelector('nav');
+
+parentNav.prepend(firstContent);
+parentNav.appendChild(lastContent);
+
+const navColor = document.querySelectorAll("nav a");
+navColor.forEach(element => {
+  element.style.color = "#008000";
+});
+
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// CTA -------------------------------------------
+const ctaTitle = document.querySelector(".cta-text h1");
+ctaTitle.style.whiteSpace = 'pre';
+ctaTitle.textContent = "DOM \n Is \n Awesome";
+
+const ctaButton = document.querySelector(".cta-text button");
+ctaButton.textContent = siteContent["cta"]["button"];
+
+const ctaImg = document.getElementById("cta-img");
+ctaImg.setAttribute('src', siteContent["cta"]["img-src"])
+
+
+// Main Content-------------------------------------
+
+
+const topTitle1 = document.querySelectorAll(".top-content .text-content h4")[0];
+topTitle1.textContent = siteContent["main-content"]["features-h4"];
+
+const topContent1 = document.querySelectorAll(".top-content .text-content p")[0];
+topContent1.textContent = siteContent["main-content"]["features-content"];
+
+
+const topTitle2 = document.querySelectorAll(".top-content .text-content h4")[1];
+topTitle2.textContent = siteContent["main-content"]["about-h4"];
+
+const topContent2 = document.querySelectorAll(".top-content .text-content p")[1];
+topContent2.textContent = siteContent["main-content"]["about-content"];
+
+let midImg = document.getElementById("middle-img");
+midImg.setAttribute('src', siteContent["main-content"]["middle-img-src"])
+
+const bottomTitle1 = document.querySelectorAll(".bottom-content .text-content h4")[0];
+bottomTitle1.textContent = siteContent["main-content"]["services-h4"];
+
+const bottomContent1 = document.querySelectorAll(".bottom-content .text-content p")[0];
+bottomContent1.textContent = siteContent["main-content"]["services-content"];
+
+const bottomTitle2 = document.querySelectorAll(".bottom-content .text-content h4")[1];
+bottomTitle2.textContent = siteContent["main-content"]["product-h4"];
+
+const bottomContent2 = document.querySelectorAll(".bottom-content .text-content p")[1];
+bottomContent2.textContent = siteContent["main-content"]["product-content"];
+
+const bottomTitle3 = document.querySelectorAll(".bottom-content .text-content h4")[2];
+bottomTitle3.textContent = siteContent["main-content"]["vision-h4"];
+
+const bottomContent3 = document.querySelectorAll(".bottom-content .text-content p")[2];
+bottomContent3.textContent = siteContent["main-content"]["vision-content"];
+
```

Comment:
There's nothing wrong here, you could group them as like to make it a little easier to read